crates/libduckdb-sys/build.rs

```diff
@@ -265,8 +301,7 @@ mod build_linked {
         // See if pkg-config can do everything for us.
         match pkg_config::Config::new().print_system_libs(false).probe(link_lib) {
             Ok(mut lib) => {
-                if let Some(mut header) = lib.include_paths.pop() {
-                    header.push("duckdb.h");
+                if let Some(header) = lib.include_paths.pop() {
```

Comment:
this looks suspicious. noticed i had it in my vendored version but not sure its correct as i don't think i ever exercise the codepath